src/translate_c/ast.zig

```diff
@@ -799,6 +804,7 @@ const Context = struct {
     }
 
     fn addIdentifier(c: *Context, bytes: []const u8) Allocator.Error!TokenIndex {
+        if (std.mem.eql(u8, bytes, "_")) return addTokenFmt(c, .identifier, "@\"_\"", .{});
```

Comment:
`std/zig/fmt.zig` defines a `isValidId` function which would be the other place this could be implemented; but that seems like it could have far-ranging consequences. If it's safe to make the change there though I can remove this and put the check there.

Reply:
`isValidId` doesn't seem to be used outside `formatId` so I think it would be safe to change there.